src/ServiceManager/Travely.ServiceManager.Service/Services/ActivityService.cs

```diff
@@ -0,0 +1,58 @@
+﻿using Grpc.Core;
+using System.Threading.Tasks;
+
+namespace Travely.ServiceManager.Service
+{
+    public class ActivityService : ActivityProto.ActivityProtoBase
+    {
+        private readonly IActivityManager _activityManager;
+
+        public ActivityService(IActivityManager activityManager)
+        {
+            _activityManager = activityManager;
+        }
+        public override async Task<ActivityResponce> CreateActivity(Activity activity, ServerCallContext context)
```

Comment:
Please pay attention to styling. Here a new line above is missing.

Reply:
reviewed every file, did some corrections